pkg/antctl/raw/traceflow/command_test.go

```diff
@@ -153,3 +228,524 @@ func TestParseFlow(t *testing.T) {
+					Name: "dummy-traceflow-dst-Pod",
+				},
+				Spec: v1alpha1.TraceflowSpec{
+					Destination: v1alpha1.Destination{
+						Namespace: "default",
+						Pod:       "Pod-2",
+					},
+				},
+			},
+		},
+		{
+			name: "dummy-traceflow-src-Pod",
+			src:  srcPod,
+			expectedTf: v1alpha1.Traceflow{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "dummy-traceflow-src-Pod",
+				},
+				Spec: v1alpha1.TraceflowSpec{
+					Source: v1alpha1.Source{
+						Namespace: "default",
+						Pod:       "Pod-1",
+					},
+				},
+			},
+		},
+		{
+			name: "dummy-traceflow-Pod-to-IP",
+			src:  "Pod-1",
+			dst:  IP,
+			expectedTf: v1alpha1.Traceflow{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "dummy-traceflow-Pod-to-IP",
+				},
+				Spec: v1alpha1.TraceflowSpec{
+					Source: v1alpha1.Source{
+						Namespace: "default",
+						Pod:       "Pod-1",
+					},
+					Destination: v1alpha1.Destination{
+						IP: IP,
+					},
+				},
+			},
+		},
+		{
+			name:        "dummy-traceflow-IP-to-Pod-with-liveTraffic",
+			src:         IP,
+			dst:         "Pod-2",
+			liveTraffic: true,
+			expectedTf: v1alpha1.Traceflow{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "dummy-traceflow-IP-to-Pod-with-liveTraffic",
+				},
+				Spec: v1alpha1.TraceflowSpec{
+					Source: v1alpha1.Source{
+						IP: IP,
+					},
+					Destination: v1alpha1.Destination{
+						Namespace: "default",
+						Pod:       "Pod-2",
+					},
+					LiveTraffic: true,
+				},
+			},
+		},
+		{
+			name: "dummy-traceflow-Pod-to-Service",
+			src:  srcPod,
+			dst:  "Service",
+			expectedTf: v1alpha1.Traceflow{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "dummy-traceflow-Pod-to-Service",
+				},
+				Spec: v1alpha1.TraceflowSpec{
+					Source: v1alpha1.Source{
+						Namespace: "default",
+						Pod:       "Pod-1",
+					},
+					Destination: v1alpha1.Destination{
+						Namespace: "default",
+						Service:   "Service",
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			_, localOpt := newCommandAndOptions(tc.src, tc.dst, "yaml", tc.timeout, tc.liveTraffic, tc.droppedOnly, false)
+			option = localOpt
+			getTFName = func(prefix string) string {
+				return tc.name
+			}
+			tf, err := getNewTraceflow(k8sClient)
+			assert.NoError(t, err)
+			assert.Equal(t, &tc.expectedTf, tf)
+		})
+	}
+}
+
+func TestGetTfName(t *testing.T) {
+
+	tests := []struct {
+		name   string
+		prefix string
+		nowait bool
+	}{
+		{
+			name:   "nowait is true",
+			prefix: "traceflow",
+			nowait: true,
+		},
+		{
+			name:   "nowait is false",
+			prefix: "traceflow",
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			_, localOpt := newCommandAndOptions("", "", "", defaultTimeout, false, false, tc.nowait)
+			option = localOpt
+			got := getTfName(tc.prefix)
+			if tc.nowait {
+				assert.Equal(t, tc.prefix, got)
+			} else {
+				assert.Regexp(t, fmt.Sprintf("^%s-........$", tc.prefix), got)
```

Comment:
Probably just use `assert.Contains`

Reply:
done, previously it was more tightly coupled.

Reply:
```suggestion
				assert.Regexp(t, fmt.Sprintf("^%s-.{8}$", tc.prefix), got)
```